Makefile

```diff
@@ -16,6 +16,7 @@ run:
 	go run main.go
 test:
 	go test ./...
+	go test ./... -json | go-test-report 
```

Comment:
Aren't you running `go test` twice with this?

Reply:
yes...will fix it